tests/Paysera/WalletApi/MapperTest.php

```diff
@@ -114,4 +128,188 @@ public function testDecodesPep()
+                'input' => [
+                    'wallet' => [
+                        'id' => 1,
+                        'owner' => null,
+                        'account' => [
+                            'number' => 'EVP1'
+                        ]
+                    ],
+                    'scopes' => [],
+                ],
+                'expected' => Paysera_WalletApi_Entity_ClientPermissionsToWallet::create()
+                    ->setWallet($this->getWallet(1, 'EVP1'))
+                    ->setScopes([]),
+            ],
+            'Case 2 Balance only' => [
+                'input' => [
+                    'wallet' => [
+                        'id' => 2,
+                        'owner' => null,
+                        'account' => [
+                            'number' => 'EVP2'
+                        ]
+                    ],
+                    'scopes' => [
+                        'balance'
+                    ],
+                ],
+                'expected' => Paysera_WalletApi_Entity_ClientPermissionsToWallet::create()
+                    ->setWallet($this->getWallet(2, 'EVP2'))
+                    ->setScopes([Paysera_WalletApi_OAuth_Consumer::SCOPE_BALANCE]),
+            ],
+            'Case 3 Statements only' => [
+                'input' => [
+                    'wallet' => [
+                        'id' => 3,
+                        'owner' => null,
+                        'account' => [
+                            'number' => 'EVP3'
+                        ]
+                    ],
+                    'scopes' => [
+                        'statements'
+                    ],
+                ],
+                'expected' => Paysera_WalletApi_Entity_ClientPermissionsToWallet::create()
+                    ->setWallet($this->getWallet(3, 'EVP3'))
+                    ->setScopes([Paysera_WalletApi_OAuth_Consumer::SCOPE_STATEMENTS]),
+            ],
+            'Case 4 Both balance and statements' => [
+                'input' => [
+                    'wallet' => [
+                        'id' => 4,
+                        'owner' => null,
+                        'account' => [
+                            'number' => 'EVP4'
+                        ]
+                    ],
+                    'scopes' => [
+                        'balance',
+                        'statements',
+                    ],
+                ],
+                'expected' => Paysera_WalletApi_Entity_ClientPermissionsToWallet::create()
+                    ->setWallet($this->getWallet(4, 'EVP4'))
+                    ->setScopes([
+                        Paysera_WalletApi_OAuth_Consumer::SCOPE_BALANCE,
+                        Paysera_WalletApi_OAuth_Consumer::SCOPE_STATEMENTS,
+                    ]),
+            ],
+            'Case 5 Unknown scope' => [
+                'input' => [
+                    'wallet' => [
+                        'id' => 5,
+                        'owner' => null,
```

Comment:
I see in each test 'owner' is null, do we even need that value? if yes, maybe would be also good to test if it's not null.

Reply:
In my case I no need this value, but here https://github.com/paysera/lib-wallet-php-client/blob/a574f2eb95374b4ddae1d8d9a03b694558dda3c4/src/Paysera/WalletApi/Mapper.php#L1206 I'll get an error if not provide this key.

Reply:
this means it must be something like owner1 if it's string or 1 if int, by the way, that's the point, that I think we are missing owner.

Reply:
Actually we haven't owner info in this point and it's unnecessary information. But if I provide some id like 1 that means that wallet belongs to client with id 1. It could cause unpredictable behavior and bugs.

Reply:
In real situation there can't be wallet without owner

Reply:
In my case I need only wallet ID as I resolved wallet by ID on wallet API side.